ukvm/ukvm_module_net.c

```diff
@@ -178,8 +178,10 @@ static void hypercall_netwrite(struct ukvm_hv *hv, ukvm_gpa_t gpa)
     int ret;
 
     ret = write(netfd, UKVM_CHECKED_GPA_P(hv, wr->data, wr->len), wr->len);
-    assert(wr->len == ret);
-    wr->ret = 0;
+    if (wr->len == ret)
+        wr->ret = 0;
+    else
+        wr->ret = -1;
 }
```

Comment:
@mato I think this is the most verbose thread about error handling in ukvm (/solo5) -- could we move it into a separate issue (since discussions on merged PRs is slightly annoying) -- #174 seems to be related as well.